.flowconfig

```diff
@@ -1,5 +1,5 @@
 [ignore]
-node_modules
+.*/node_modules/.*
```

Comment:
This breaks module resolution in my editor:

![Screen Shot 2020-12-08 at 4 53 53 PM](https://user-images.githubusercontent.com/1587282/101546500-dce74400-3976-11eb-8f60-72b39f0f1481.png)

What's the goal in this change?

Reply:
Oh, interesting.  I added that change because flow was linting code inside node_modules and failing on my local.

Reply:
I believe I added the same in `vip-ui`.
Notably, I also started to notice some module resolution issues in my editor as well, but it seemed unpredictable when.

Reply:
I'll pull it out for now -- let's revisit in another PR if it's still failing for you.